docs/source/checks/tabular/model_evaluation/plot_unused_features.py

```diff
@@ -50,10 +50,10 @@
 from deepchecks.tabular.checks import UnusedFeatures
 from deepchecks.tabular.datasets.classification import adult
 
-_, test_ds = adult.load_data()
+train_ds, test_ds = adult.load_data()
 model = adult.load_fitted_model()
 
-result = UnusedFeatures(feature_variance_threshold=1.5).run(test_ds, model)
+result = UnusedFeatures(feature_variance_threshold=1.5).run(train_ds, test_ds, model)
```

Comment:
UnusedFeatures is a SingleDatasetCheck, so no need to add a second dataset for it